stable/xray/templates/xray-statefulset.yaml

```diff
@@ -481,6 +500,9 @@ spec:
         volumeMounts:
         - name: data-volume
           mountPath: "{{ .Values.xray.persistence.mountPath }}"
+{{- if .Values.common.customVolumeMounts }}
```

Comment:
Please use a with statement for these blocks, it is much cleaner than repeating the value twice, plus it removes the need for the if statement

Reply:
Done